viam-cartographer.go

```diff
@@ -509,6 +514,24 @@ func (cartoSvc *cartographerService) GetPosition(ctx context.Context) (spatialma
 	return vcUtils.CheckQuaternionFromClientAlgo(pose, componentReference, returnedExt)
 }
 
+func (cartoSvc *cartographerService) getPositionModularizationV2(ctx context.Context) (spatialmath.Pose, string, error) {
+	pos, err := cartoSvc.cartofacade.GetPosition(ctx, cartoSvc.cartoFacadeTimeout)
+	if err != nil {
+		return spatialmath.NewPose(r3.Vector{}, spatialmath.NewOrientationVector()), "", err
```

Comment:
why not:
```
return spatialmath.NewZeroPose(), "", err
```

Reply:
I must've missed that function - udpdated!